docs/tutorials/podman_pasta_ipv6_with_snid_ipv4.md

```diff
@@ -0,0 +1,761 @@
+```
+
+## Compose with Port Mapping + Minimal Pasta Line
+With this Method, `podman ps` will show the Open Ports in its Output.
+
+```
+services:
+  whoami-caddy:
+    image: caddy:latest
+    #image: lucaslorentz/caddy-docker-proxy:2.9-alpine
+    pull_policy: "missing"
+    container_name: whoami-caddy
+    restart: "unless-stopped"
+    security_opt:
+      - no-new-privileges:true
+      - label=type:container_runtime_t
+    ports:
+      - target: 80
+        host_ip: "[${APPLICATION_IPV6_ADDRESS}]"
+        published: 80
+        protocol: tcp
+      - target: 443
+        host_ip: "[${APPLICATION_IPV6_ADDRESS}]"
+        published: 443
+        protocol: tcp
+      - target: 443
+        host_ip: "[${APPLICATION_IPV6_ADDRESS}]"
+        published: 443
+        protocol: udp
+    network_mode: "pasta:--ipv6-only"
+    volumes:
+ #     - /run/user/1001/podman/podman.sock:/var/run/docker.sock:rw,z
+      - ./Caddyfile:/etc/caddy/Caddyfile:ro,z
+      - ~/containers/data/whoami-caddy:/data:rw,z
+      - ~/containers/log/whoami-caddy:/var/log:rw,z
+      - ~/containers/config/whoami-caddy:/config:rw,z
+      - ~/containers/certificates/letsencrypt:/certificates:ro,z
+    environment:
+      - CADDY_DOCKER_CADDYFILE_PATH=/etc/caddy/Caddyfile
+
+  # Proxy to container
+  whoami-application:
+    image: traefik/whoami
+    pull_policy: "missing"
+    container_name: whoami-application
+    restart: "unless-stopped"
+    network_mode: "service:whoami-caddy"
+    environment:
+      - WHOAMI_PORT_NUMBER=8080
+```
+
+## Compose with only Pasta Line
+With this Method, `podman ps` will NOT show the Open Ports in its Output.
+
+In fact, `podman inspect <container>` will NOT list the IP Addresses nor where the Ports are bound to.
+
+This works correctly, but the only way to examine if the Port is used is to run `ss -nlt6`.
+
+```
+services:
+  whoami-caddy:
+    image: caddy:latest
+    #image: lucaslorentz/caddy-docker-proxy:2.9-alpine
+    pull_policy: "missing"
+    container_name: whoami-caddy
+    restart: "unless-stopped"
+    security_opt:
+      - no-new-privileges:true
+      - label=type:container_runtime_t
+    network_mode: "pasta:--ipv6-only,-t,${APPLICATION_IPV6_ADDRESS}/80,-t,${APPLICATION_IPV6_ADDRESS}/443,-u,${APPLICATION_IPV6_ADDRESS}/443"
+    volumes:
+ #     - /run/user/1001/podman/podman.sock:/var/run/docker.sock:rw,z
+      - ./Caddyfile:/etc/caddy/Caddyfile:ro,z
+      - ~/containers/data/whoami-caddy:/data:rw,z
+      - ~/containers/log/whoami-caddy:/var/log:rw,z
+      - ~/containers/config/whoami-caddy:/config:rw,z
+      - ~/containers/certificates/letsencrypt:/certificates:ro,z
+    environment:
+      - CADDY_DOCKER_CADDYFILE_PATH=/etc/caddy/Caddyfile
+
+  # Proxy to container
+  whoami-application:
+    image: traefik/whoami
+    pull_policy: "missing"
+    container_name: whoami-application
+    restart: "unless-stopped"
+    network_mode: "service:whoami-caddy"
+    environment:
+      - WHOAMI_PORT_NUMBER=8080
+```
```

Comment:
unnecessary. i think the other version should be preferred and this one dropped completely.

Reply:
Merged the two `compose.yml` together.

Excerpt of `compose.yml` showing both possibilities:
```
# Ports Section (DISABLE if using the Pasta Extended Line)
    ports:
      - target: 80
        host_ip: "[${APPLICATION_IPV6_ADDRESS}]"
        published: 80
        protocol: tcp
      - target: 443
        host_ip: "[${APPLICATION_IPV6_ADDRESS}]"
        published: 443
        protocol: tcp
      - target: 443
        host_ip: "[${APPLICATION_IPV6_ADDRESS}]"
        published: 443
        protocol: udp
    # Pasta Minimal Line (ENABLE the Normal Ports Section above if using this Method)
    network_mode: "pasta:--ipv6-only"
    # Pasta Extended Line (DISABLE the Normal Ports Section above if using this Method)
    # network_mode: "pasta:--ipv6-only,-t,${APPLICATION_IPV6_ADDRESS}/80,-t,${APPLICATION_IPV6_ADDRESS}/443,-u,${APPLICATION_IPV6_ADDRESS}/443"
```

And small Note about the Impact:
```
Note that:
- When using the Normal Ports Section and the Pasta Minimal Line, `podman ps` will show the Open Ports in its Output.
- When using the Pasta Extended Line, `podman ps` will NOT show the Open Ports in its Output. In fact, `podman inspect <container>` will NOT list the IP Addresses nor where the Ports are bound to. This works correctly, but the only way to examine if the Port is used is to run `ss -nlt6`.
```

Reply:
Closing. Please re-open or open a new Conversation if not satisfactory.